src/decenc_openjpeg.c

```diff
@@ -512,4 +584,4 @@ enc_jpeg2000(unsigned char *cin, g2int width, g2int height, g2int nbits,
     return iret;
 }
 
-#endif
+#endif
```

Comment:
Add back a new-line character at the end of the last line. Here, and in other three files touched by this PR.